clients/banking/src/components/TransferInternationalWizardDetails.tsx

```diff
@@ -111,15 +111,15 @@ export const TransferInternationalWizardDetails = ({
     .with(AsyncData.P.Done(Result.P.Ok(P.select())), ({ fields }) => (
       <View>
         <Tile>
-          <View {...(isLoading && animations.heartbeat)}>
+          <TransitionView {...(isLoading && animations.heartbeat)}>
```

Comment:
same here, it should remain a `View`

Reply:
I didn't had animations with the `View`. Maybe because this is using keyframes? Those does not work with inline style.